.github/workflows/build_and_test.yml

```diff
@@ -578,19 +578,15 @@ jobs:
     strategy:
       matrix:
         include:
-          - os: "windows-2019"
-            platform: "Visual Studio 16 2019"
-            arch: "x64"
-            pcap_lib: "npcap"
-          - os: windows-2019
-            platform: "Visual Studio 16 2019"
+          - os: windows-2025
+            platform: "Visual Studio 17 2022"
             arch: Win32
             pcap_lib: "winpcap"
```

Comment:
Since WinPcap is deprecated I think we should have 2 build running with Npcap and one with WinPcap.
I'd suggest having this build running with Npcap